app/assets/stylesheets/show.scss

```diff
@@ -156,7 +156,7 @@
 
   .header {
     font-weight: 700;
-    font-size: large;
+    font-size: x-large;
```

Comment:
Bumping a couple font sizes now that Argo is opinionated about fonts. Prior to this PR, Argo appeared to be fine with any of a grab-bag of fonts from Blacklight's defaults, which allow browsers to vary the display wildly.